test/fixtures/identical-file-runner.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path')
+const assert = require('assert')
+const file1 = require('./identical-file1.js')
+const file2 = require('./identical-file2.js')
+
+assert.equal(file1(), file2())
+
+const cov = (new Function('return this.__coverage__'))()
```

Comment:
I'm open to better ways to test that the coverage data includes the 3 files.

Reply:
@coreyfarrell this assertion seems reasonable to me _but_, out of curiosity, what would the behavior have been before your fixes to the caching transform, would we have only had one of the two files listed in the coverage object?

mind testing running this test prior to your fix and confirming that it breaks as expected?

Reply:
I did test it, without the fixes to caching-transform this test would not record results for identical-file2.js.  This is because both identical files have the same hash value so caching-transform gave the instrumented code from file1 to file2.  This meant file2 recorded it's results to file1.